pkg/controller/restore_controller.go

```diff
@@ -324,6 +326,22 @@ func (c *restoreController) validateAndComplete(restore *api.Restore, pluginMana
 		return backupInfo{}
 	}
 
+	// validate Restore Init Hook's InitContainers
+	restoreHooks, err := hook.GetRestoreHooksFromSpec(&restore.Spec.Hooks)
+	if err != nil {
+		restore.Status.ValidationErrors = append(restore.Status.ValidationErrors, err.Error())
+	}
+	for _, resource := range restoreHooks {
```

Comment:
risk of nil pointer when error happens in `GetRestoreHooksFromSpec`

Reply:
restoreHooks's type is []ResourceRestoreHook.
If `GetRestoreHooksFromSpec` returns an empty array, I think this loop will not be entered, so I think there should be no nil pointer risk.

Reply:
It does return nil
https://github.com/vmware-tanzu/velero/blob/52504b548dd63d249f7ebceebc6a8c9ab95b4a66/internal/hook/item_hook_handler.go#L401

But you are right it won't panic.

Reply:
Got it. I will change return value to empty array.